### ISSUE TITLE
Jahresbeleg mit ausgefallener Signaturerstellungseinheit erstell!

### ISSUE THREAD

**Issue by alexrksv** [BMF-RKSV-Technik/at-registrierkassen-mustercode#649]
Liebe Kollegen,

es wurde ein Jahresbeleg mit ausgefallener Signaturerstellungseinheit erstellt. Jetzt kommt es auf Finanzonline die Fehlermeldung, dassdie Formatierung des Registrierkassenalgorithmuskennzeichens im maschinenlesbaren COde des vorliegenden Beleges einen Fehler aufweist.

Wie ist in so einem Fall vorzugehen ?

Danke vorab für die Informationen.

**Comment by ErichFreitag** [comment]
Die Fehlermeldung lautet "Reg.Alg.-Kennzeichen ist fehlerhaft"?

Ich kenne diese Meldungen "mit ausgefallener SE-Einheit", obwohl der maschinenlesbare Code selbst korrekt ist. Es scheint, dass der Status der SE-Einheit zum Zeitpunkt der Belegserstellung Vorrang gegenüber dem Beleginhalt selbst hat.

Ob das seitens FON noch geändert wird ist mir derzeit noch nicht bekannt. Sie können aber raschest möglich einen neuen Jahresbeleg oder Nullbeleg erzeugen und diesen nochmals prüfen. Sicherheitshalber würde ich auch den Erstversuch dokumentieren.

**Comment by alexrksv** [comment]
@ErichFreitag danke für die schnelle Antwort. Ich werde mit einem Nullbeleg probieren. 

Aber hat jemand andere eventuell versucht einen Jahresbeleg mit ausgefallener Signaturerstellungseinheit zu prüfen ? Würde mich auf eure Erfahrungen sehr freuen.

**Comment by ErichFreitag** [comment]
Ja, ich habe mehrere Fälle, wo Jahresbelege mit ausgefallener SE-Einheit geprüft und abgelehnt wurden. Die Fehlermeldung war nur missverständlich dabei "Signatur = ausgefallen" obwohl diese zu diesem Zeitpunkt korrekt war. 

Das passiert mit HSM-Anbindungen, wo eine Kasse "failed" meldet und die anderen Kassen danach Jahresbelege mit funktionierender SE-Einheit generiert haben, die vorherige Kasse die SE-Einheit aber noch nicht wieder OK gemeldet hat.

**Comment by AxelKutschera** [comment]
@ErichFreitag Du meinst damit den Status der SE bei FON.

**Comment by ErichFreitag** [comment]
Korrekt, Axel.

**Comment by mandihof** [comment]
Hallo, ja ich hab das Problem auch. Hab einige Kunden bei denen der stick nicht angesteckt war oder nicht erkannt wurde... jetzt haben wir jahresbelege mit ausgefallener signatureinheut die das fon nicht akzeptiert... toll...

Aber ich kann ja nicht 2 wochen und zb 20 Belege später einen neuen jahresbelege (nullbeleg) melden, oder?

**Comment by ErichFreitag** [comment]
Doch, das können und sollen sie auch. Es ist explizit vom BMF so kommuniziert, dass die Jahresbelege mit funktionierender SE-Einheit bis spätestens 15.02. nachzuholen sind. Gegebenenfalls erzeugen sie dazu "normale" Nullbelege.

**Comment by AxelKutschera** [comment]
"nachzuholen" bedeutet nach Wunsch des BMF "über FON positiv geprüft".

**Comment by mandihof** [comment]
Okay super. Danke, dann machen wir das so
